index.ts

```diff
@@ -21,8 +21,8 @@ const run = async (): Promise<void> => {
     const buildPath = getInput('build_path');
     const moropoUrl = new URL(getInput('moropo_url'));
     const moropoApiUrl = new URL(getInput('moropo_api_url'));
-    const githubPersonalAccessToken = new URL(getInput('github_access_token'));
-    const sync = new URL(getInput('sync'));
+    const githubPersonalAccessToken = getInput('github_access_token');
```

Comment:
wouldn't this be passed in in place of github_token?

Reply:
yeah I'll refactor that when I get it working